docs/virtual_machines/virtual_machine_instances.md

```diff
@@ -67,10 +67,10 @@ require a provisioner in your cluster.
 ## Additional Information
 
 -   More information about persistent and ephemeral volumes:
-    [Disks and Volumes](creation/disks-and-volumes.md)
+    [Disks and Volumes](disks_and_volumes.md)
 
 -   How to access a VirtualMachineInstance via `console` or `vnc`:
-    [Console Access](usage/graphical-and-console-access.md)
+    [Console Access](graphical_and_console_access.md)
 
 -   How to customize VirtualMachineInstances with `cloud-init`:
```

Comment:
This link is dead

Reply:
Investigating

Reply:
OK, I see what's happenin' here ... We have half of an internal link and half an external link.  The two halves are creating the failure.

We have 19 cases of this in the user-guide.  I'm going to update them all.

Stand by.

Reply:
Link is fixed per local test and Netlify